media/css/products/vpn/landing.scss

```diff
@@ -110,6 +110,15 @@ $image-path: '/media/protocol/img';
     margin-top: $layout-lg;
 }
 
+.vpn-hero-secondary {
+    margin-top: $spacing-lg;
+    @include text-body-md;
+}
+
+.vpn-hero-guarantee {
+    @include text-body-sm;
+}
+
```

Comment:
These don't actually apply to the landing page. Should I clone this file and setup a seo-specific style bundle?

Reply:
The design for the landing page is changing as of next week, so I would try to avoid depending on it. It's a little unfortunate that these pages are going to be outdated almost straight away, but if getting things out the door is more important for SEO then we should try and avoid dependencies like this.

Reply:
I duplicated this file into the platform directory/bundle and reverted these changes.